fabtests/Makefile.am

```diff
@@ -362,6 +363,10 @@ functional_fi_bw_SOURCES = \
 	functional/bw.c
 functional_fi_bw_LDADD = libfabtests.la
 
+functional_fi_flood_SOURCES = \
```

Comment:
Also update the windows build files as well - fabtests.vcxproj and fabtests.vcxproj.filters